conf/deployuidb.sh

```
@@ -7,24 +7,59 @@ host="$3"
 port="$4"
 zstack_ui_db_password="$5"
 
+MYSQL='mysql'
+
+if [[ `id -u` -ne 0 ]] && [[ x"$user" = x"root" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)